### PR TITLE
MOB-3333 [Config & Prep] Set the default map zoom

### PR DESCRIPTION
**Pull request by mateid-kr**
## JIRA
https://karhoo.atlassian.net/browse/MOB-3333

## DESCRIPTION
MOB-3333 [Config & Prep] Set the default map zoom

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [X] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [X] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

**Review by cornelkar — DISMISSED** [review]

File: KarhooUISDK/Screens/AddressScreen/AddressViewController.swift [hunk 1]
```diff
@@ -157,12 +157,7 @@ final class AddressViewController: UIViewController, AddressView {
     }
 
     private func configureForAuthenticationMethod() {
-        if Karhoo.configuration.authenticationMethod().guestSettings == nil {
-            buildAddressMapView()
-        } else {
-            setOnMapView.isHidden = true
-            currentLocationView.isHidden = true
-        }
+        buildAddressMapView()
```
Comment (mohadian) [inline KarhooUISDK/Screens/AddressScreen/AddressViewController.swift:160]: Is it possible in iOS to check if the application has access to Location service or not? 

If the application is denided to access the location, we need to hide the currentLocationView and setOnMapView (similar to Android)

```
setOnMapView.isHidden = false
currentLocationView.isHidden = false
```

File: KarhooUISDK/Screens/BookingScreen/Map/PickupOnlyStrategy.swift [hunk 2]
```diff
@@ -102,11 +95,8 @@ final class PickupOnlyStrategy: PickupOnlyStrategyProtocol, BookingMapStrategy,
             }
         }
 
-        if Karhoo.configuration.authenticationMethod().isGuest() {
-            focousOnPickup()
```
Comment (mohadian) [inline KarhooUISDK/Screens/BookingScreen/Map/PickupOnlyStrategy.swift:106]: the `focousOnPickup` seems no longer needed? 

Is it possible to check if we have location permission? if we dont have location permission, perhaps we can display a dialog asking user to go to settings and enable the location permission?

Comment (mohadian) [reply KarhooUISDK/Screens/BookingScreen/Map/PickupOnlyStrategy.swift:106]: If it is not achievable to do it as part of this task, you can add a ticket to tackle it separately 

Here is an example of the dialog in Google Maps

Screenshot 2021-05-26 at 16 37 15

Turn on your location settings

Comment (mateid-kr) [reply KarhooUISDK/Screens/BookingScreen/Map/PickupOnlyStrategy.swift:106]: Added back the focus on pickup. For the popup, we should have a different ticket

**Review by mohadian — APPROVED** [review]

**Review by cornelkar — APPROVED** [review]

**Pull request merged**